scripts/setup.py

```diff
@@ -414,4 +428,9 @@ def validateParameters(project):
     ios.run()
     flutter = Flutter(project)
     flutter.run()
+    print('🤖 Generating project...')
+    move_project_to_root(project)
+    print('Would you like to remove `.template` and `.github` folder? (y/n)')
+    if input().lower() == 'y':
+        clean_up()
```

Comment:
Template code cleaning up should be a default task I think 🤔

Reply:
@luongvo I'm not sure because if removing `.template`, we are not able to do it again, so I left an option for now. It could change in the future 😁

Reply:
Because the first step is cloning this repo so I think the user could easily reset the history to re-run it. Also, it's not a normal thing we will do often so... ~~ 😄

Reply:
@luongvo haha, okay. It's easy to update 😋

Reply:
you awesome :hug